packages/experimental/Avatar/src/useAvatar.ts

```diff
@@ -43,6 +46,8 @@ export const useAvatar = (props: JSAvatarProps): AvatarInfo => {
       image: imageProps,
       badge: badgeProps,
       initials: _initials,
+      accessibilityLabel: accessibilityText,
```

Comment:
Should we prioritize the accessibilityLabel here similar to how we do on line 21? (i.e. if someone sets an accessibilityLabel we should use that over the default accessibilityLabel we set for them with accessibilityText)